tests/sm_int/sm_int.c

```diff
@@ -17,7 +17,7 @@
 #include "azure_macro_utils/macro_utils.h"
 
 #include "azure_c_pal/timer.h"
-#include "azure_c_pal/interlocked_hl.h"
+#include "azure_c_util/interlocked_hl.h"
```

Comment:
>interlocked_hl [](start = 23, length = 14)

not used here anyway?

Reply:
Removed

---
In reply to: [501164641](https://github.com/Azure/azure-c-util/pull/66#discussion_r501164641) [](ancestors = 501164641)